backend/test/mock_audits/cs_audit.html

```diff
@@ -1,19 +1,29 @@
-
 <!-- saved from url=(0108)https://prod-web.neu.edu/wasapp/DARSStudent/ParseFriendlyServlet?DETAILS=2019061313074997&int_seq_no=6659805 -->
-<html><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252"><title>Web Audit</title>
- <style type="text/css">
- <!--
-  body, a {font-family: arial, helvetica, sans-serif}
-  h1 {font-family: arial, helvetica, sans-serif; font-size: x-large; pause: 200% 150%}
-  h2 {font-family: arial, helvetica, sans-serif; font-size: large; pause: 200% 150%}
-  h3 {font-family: arial, helvetica, sans-serif; font-size: medium; pause: 200% 150%}
-  .auditText{font-family: Courier, Courier New, mono;}
-  .auditPreviewText{text-family: courier, courier new, mono; color:black; font-size:10pt;}
- -->
- </style>
-</head><body>
-<a class="auditPreviewText" href="https://prod-web.neu.edu/wasapp/DARSStudent/ParseAuditServlet?job_id=2019061313074997&amp;int_seq_no=6659805">Back</a>
-<pre><font class="auditPreviewText">PREPARED: 06/13/19 - 13:07                          001479347</font>
+<html>
+    <head>
+        <meta
+            http-equiv="Content-Type"
+            content="text/html; charset=windows-1252"
+        />
+        <title>Web Audit</title>
+        <style type="text/css">
+            <!--
+             body, a {font-family: arial, helvetica, sans-serif}
+             h1 {font-family: arial, helvetica, sans-serif; font-size: x-large; pause: 200% 150%}
+             h2 {font-family: arial, helvetica, sans-serif; font-size: large; pause: 200% 150%}
+             h3 {font-family: arial, helvetica, sans-serif; font-size: medium; pause: 200% 150%}
+             .auditText{font-family: Courier, Courier New, mono;}
+             .auditPreviewText{text-family: courier, courier new, mono; color:black; font-size:10pt;}
+            -->
+        </style>
+    </head>
+    <body>
+        <a
+            class="auditPreviewText"
+            href="https://prod-web.neu.edu/wasapp/DARSStudent/ParseAuditServlet?job_id=2019061313074997&amp;int_seq_no=6659805"
+            >Back</a
+        >
+        <pre><font class="auditPreviewText">PREPARED: 06/13/19 - 13:07                          001479347</font>
```

Comment:
I don't think reformatting the audits will change the test results, but we can check later today.